components/collection/drop/CollectedBy.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div
+    class="rounded-full h-[62px] md:w-auto border border-k-shade inline-flex items-center justify-between px-3">
+    <div class="flex items-center">
+      <CollectionDropCollectedByAvatar
+        v-for="(address, index) in addresses.slice(0, maxAddressCount)"
+        :key="address"
+        :class="[zIndexMap[index], { '-ml-2': index > 0 }]"
+        :address="address" />
+    </div>
+
+    <div v-if="remainingCount" class="ml-2.5">+{{ remainingCount }}</div>
+  </div>
+</template>
+
+<script setup lang="ts">
+const props = defineProps<{
+  addresses: string[]
+}>()
+
+const maxAddressCount = 5
+const zIndexMap = ['z-50', 'z-40', 'z-30', 'z-20', 'z-10']
```

Comment:
The `z-index` could be computed value instead of hard coding.

```
v-for="(address, index) in addresses.slice(0, maxAddressCount)"

:class="z-{ maxAddressCount - index }"
```

Reply:
it‘s not recommended as it prevents TailwindCSS from recognizing the class name: https://tailwindcss.com/docs/content-configuration#dynamic-class-names

<img width=300 src=https://github.com/kodadot/nft-gallery/assets/10708802/bdcc8623-6fe9-4075-bda8-f12e045c85ef />